postgres/initdb.sql

```diff
@@ -2,7 +2,6 @@
 CREATE EXTENSION postgis;
 
 CREATE TABLE IF NOT EXISTS projects (
-    archive boolean,
```

Comment:
what is the consequence of this once we deploy it? Initdb.sql will not be run when there is a database already. So we need to manually remove this column, right?

Reply:
Yes.